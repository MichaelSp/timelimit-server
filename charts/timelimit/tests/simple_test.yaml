suite: email Test Suite
templates:
- deployment.yaml
chart:
  version: 0.2.8
tests:
- it: Matches the snapshot
  asserts:
  - matchSnapshot: { }