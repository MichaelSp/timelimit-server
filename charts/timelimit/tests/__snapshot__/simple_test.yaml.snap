Matches the snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: timelimit
        app.kubernetes.io/version: 1.17.0
        helm.sh/chart: timelimit-0.2.8
      name: RELEASE-NAME-timelimit
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: timelimit
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: timelimit
        spec:
          containers:
            - env:
                - name: NODE_ENV
                  value: production
                - name: DB_DRIVER
                  value: sqlite
                - name: DB_NAME
                  value: RELEASE-NAME-timelimit
                - name: DB_USER
                  value: RELEASE-NAME-timelimit
                - name: DB_PORT
                  value: "3306"
                - name: DB_HOST
                  value: RELEASE-NAME-timelimit-mariadb
                - name: DB_PASS
                  value: ""
                - name: PORT
                  value: "8080"
                - name: ALWAYS_PRO
                  value: "no"
                - name: ADMIN_TOKEN
                  valueFrom:
                    secretKeyRef:
                      key: the-key
                      name: the-secret-name
              image: ghcr.io/michaelsp/timelimit-server/timelimit:latest
              imagePullPolicy: IfNotPresent
              livenessProbe:
                httpGet:
                  path: /time
                  port: 8080
              name: timelimit
              ports:
                - containerPort: 8080
                  name: http
                  protocol: TCP
              readinessProbe:
                httpGet:
                  path: /time
                  port: 8080
              resources: {}
              securityContext: {}
          securityContext: {}
          serviceAccountName: RELEASE-NAME-timelimit
Supports DB_DRIVER:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: timelimit
        app.kubernetes.io/version: 1.17.0
        helm.sh/chart: timelimit-0.2.8
      name: RELEASE-NAME-timelimit
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: timelimit
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: timelimit
        spec:
          containers:
            - env:
                - name: NODE_ENV
                  value: production
                - name: DB_DRIVER
                  value: mariadb
                - name: DB_NAME
                  value: RELEASE-NAME-timelimit
                - name: DB_USER
                  value: RELEASE-NAME-timelimit
                - name: DB_PORT
                  value: "3306"
                - name: DB_HOST
                  value: RELEASE-NAME-timelimit-mariadb
                - name: DB_PASS
                  value: ""
                - name: PORT
                  value: "8080"
                - name: ALWAYS_PRO
                  value: "no"
                - name: ADMIN_TOKEN
                  valueFrom:
                    secretKeyRef:
                      key: the-key
                      name: the-secret-name
              image: ghcr.io/michaelsp/timelimit-server/timelimit:latest
              imagePullPolicy: IfNotPresent
              livenessProbe:
                httpGet:
                  path: /time
                  port: 8080
              name: timelimit
              ports:
                - containerPort: 8080
                  name: http
                  protocol: TCP
              readinessProbe:
                httpGet:
                  path: /time
                  port: 8080
              resources: {}
              securityContext: {}
          securityContext: {}
          serviceAccountName: RELEASE-NAME-timelimit
